castai/resource_castai_cluster.go

```diff
@@ -100,11 +107,133 @@ func resourceCastaiCluster() *schema.Resource {
 				},
 			},
 			ClusterFieldKubeconfig: schemaKubeconfig(),
+			PolicyFieldAutoscalerPolicies: {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default: true,
+						},
+						PolicyFieldClusterLimits: {
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"enabled": {
+										Type:     schema.TypeBool,
+										Optional: true,
+										Default:  true,
+									},
+									"cpu": {
+										Type:     schema.TypeList,
+										MaxItems: 1,
+										Optional: true,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"max_cores": {
+													Type:     schema.TypeInt,
+													Optional: true,
+													Default: 21,
+												},
+												"min_cores": {
+													Type:     schema.TypeInt,
+													Optional: true,
+													Default: 2,
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+						PolicyFieldNodeDownscaler: {
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"empty_nodes": {
```

Comment:
We should declare constant for these fields for consistency, later we can discuss whether keeping constants worth it.